MapboxNavigation/CarPlayManagerDelegate.swift

```diff
@@ -152,6 +152,15 @@ public protocol CarPlayManagerDelegate {
      */
     @objc func carPlayManagerDidEndNavigation(_ carPlayManager: CarPlayManager) -> ()
     
+    /**
+     Called when the CarPlay navigation view controller detects an arrival.
+     
+     - returns: A boolean value indicating whether to display an arrival UI.
+     */
+    
+    @objc(carPlayerManager:shouldPresentArrivalUIForWaypoint:)
+    optional func carPlayManager(_ carPlayManager: CarPlayManager, shouldPresentArrivalUIFor waypoint: Waypoint) -> Bool
```

Comment:
nit: how about `shouldDisplayArrivalUIForWaypoint`?

Reply:
"present" is the verb that's being used for presenting view controllers, documents and templates which we are doing here. "display" is more for styling and when using CoreGraphics directly.